Tests/CMakeLists.txt

```diff
@@ -224,6 +224,7 @@ add_test_r(hit-2 HIT)
 add_test_r(hit-3 HIT)
 add_test_r(sod-1 Sod)
 add_test_r(sod-2 Sod)
+add_test_rv(sod-3 Sod)
```

Comment:
How about converting any test without inflow/outflow to an `add_test_rv`, for example the HIT cases? Probably could add some of the EB ones as well.

Reply:
Great, I think I did that now.